From ef7a9bbf0144d35012c214195fb0c5b727e4b942 Mon Sep 17 00:00:00 2001
From: Yousong Zhou <yszhou4tech@gmail.com>
Date: Sat, 9 Jan 2021 14:51:29 +0800
Subject: [PATCH] configure: allow enabling/disabling libudev from command line

---
 configure | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/configure b/configure
index 704aa3e76d..cc13bb1c2e 100755
--- a/configure
+++ b/configure
@@ -1527,6 +1527,10 @@ for opt do
   ;;
   --disable-fortify-source) fortify_source="no"
   ;;
+  --enable-libudev) libudev=yes
+  ;;
+  --disable-libudev) libudev=no
+  ;;
   *)
       echo "ERROR: unknown option $opt"
       echo "Try '$0 --help' for more information"
